report/test/renderer/util-test.js

```diff
@@ -33,12 +33,13 @@ describe('util helpers', () => {
   });
 
   it('builds device emulation string', () => {
-    const get = opts => Util.getEmulationDescriptions({
-      ...opts,
-      screenEmulation: {disabled: true},
-    }).deviceEmulation;
-    assert.equal(get({formFactor: 'mobile'}), 'Emulated Moto G4');
-    assert.equal(get({formFactor: 'desktop'}), 'Emulated Desktop');
+    const get = settings => Util.getEmulationDescriptions(settings).deviceEmulation;
+    /* eslint-disable max-len */
+    assert.equal(get({formFactor: 'mobile', screenEmulation: {disabled: false, mobile: true}}), 'Emulated Moto G4');
+    assert.equal(get({formFactor: 'mobile', screenEmulation: {disabled: true, mobile: true}}), 'No emulation');
```

Comment:
this does bring it back to @connorjclark's point that there's no indication this was a mobile report now

Reply:
Adding a separate indicator for that would be fine, I just don't think it makes sense here.

Reply:
https://github.com/GoogleChrome/lighthouse/issues/8178